src/components/Playlists/Playlists.js

```diff
@@ -28,7 +19,7 @@ class Playlists extends Component  {
         if(data) {
             return(
                 <div>
-                    <h1>Top 10 playlists: </h1>
+                    <h1>Top playlists: </h1>
```

Comment:
While it is prob not related to this PR but does the API actually spit out top playlists (i.e. the playlists that the user accesses the most), or just the 10 most recent ones?

Reply:
It doesn't actually say, I assume it's a combination of both (though this is just based off what I've noticed while using the app)